### DISCUSSION TITLE
Help in adding tags for a column in a table

### DISCUSSION THREAD

**Discussion by VanessaHo99** [open-metadata/OpenMetadata#18851] [Q&A]
Hi, I wanted to add tags to column in Openmetadata with REST API but have not been successful. Been using /api/v1/tags/name/{column_fqn} to try to update and retrieve column tags through POST and GET method but it results in 404 error - HTTP not found. Any help is appreciated. Thanks!

I have tried with Openmetadata ingestion but found out it was not meant to add tags to columns.

**Comment by zymxiaotie** [comment]
Same question here.
